### DISCUSSION TITLE
Welcome to ADefWebserver.Module.HtmlTextV2 Discussions!

### DISCUSSION THREAD

**Discussion by ADefWebserver** [ADefWebserver/ADefWebserver.Module.HtmlTextV2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thabaum** [comment]
Hello Oqtane community!

Look forward to helping with this project to see where we can take it.  I have a few questions and such maybe we can all get on the Oqtane Discord channel or Teams/Zoom sometime and collaborate?

**Reply by ADefWebserver** [reply]
For now it is just you and I :) Email is best for me so lets keep the discussion here (because it emails me).

**Comment by ADefWebserver** [comment]
@thabaum - Can you take a look at the latest? If I run the module in debug (using attach to process) it loads:
![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/1857799/7391ffba-88e7-452d-a9b7-547a5fafa0e5)

But if you try to run it without debugging it doesn't:
![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/1857799/f7f6a4b3-af6a-4350-b8c6-c4864c8add1f)

**Note:** you have to run Oqtane in Interactive mode for it to work at all:
![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/1857799/dad104cf-255b-494e-99b5-f834cf44c9da)

**Reply by thabaum** [reply]
@ADefWebserver I will take a look tonight start playing with it.  Thanks!

**Comment by ADefWebserver** [comment]
@thabaum - I just checked in a change that shows the editor (still with an error), but, it works on my machine without being in debug mode:

![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/1857799/38221267-8c98-455f-8e93-f2e6d4b82742)

**Reply by thabaum** [reply]
OK, Keep me up to date with any details I should have time tonight to start diving more into it.  Any error details you can share show up in the log?  Maybe need to put a try/catch block?

**Reply by thabaum** [reply]
I am wondering if something isnt loading in production that loads in debug.

**Reply by thabaum** [reply]
I reviewed your changes a bit, it will take some time to digest this project and don't have time to look at it in detail until I can get done with some other priorities.  I think maybe it might be in the way things are being passed or set initially from a quick glance.  But that is shooting in the dark with how much I have reviewed.

**Reply by ADefWebserver** [reply]
I am going to work on some other projects for now. Take your time. Let me know what you find.

**Reply by thabaum** [reply]
@ADefWebserver looking closer I believe to start we should be sure we have all dependencies up-to-date, including the framework which I can submit a PR for later.  I see .NET 8.0.4 here and 8.0.5 for framework 8.0.6 is out now.  I can submit a PR for this as well if needed but you can probably do it easier.

**Reply by thabaum** [reply]
@ADefWebserver Also be sure latest [.NET hosting bundle](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) installed if you want to try an IIS web server test. I am up to 8.0.6 going forward on my branch FYI currently.  I am just setting things up for later tonight as I have a few more things to handle today.

However I know 8.0.5 is best to at least have installed along with updating all the other dependencies as soon as things look ok going forward and on other hand may need updated to work anyhow...

Note:  I am also updating the HtmlEditor.Blazor to .Net 8 which may force me to start over but I want to see how far she goes.  I am just setting up the environment with the latest and greatest if I can.  All the other projects I have updated to 8.0.6 but need to do the framework as well.

**Reply by thabaum** [reply]
@ADefWebserver I was only able to get as far as upgrading the projects to latest .NET 8.  I was able to build in debug 8.0.6 but need to iron out an issue with the package project (release.cmd) for a release build.  That was all I had time for this weekend along with getting familiar with the project code some.

Now that I have things out of my way I can focus on this project this week starting tomorrow morning.
Cheers!

**Comment by ADefWebserver** [comment]
@thabaum - No hurry we are not on a timeline. Let me know when you are ready to PR and I can test it. Let's open a new tread when you so so. Thanks!

**Reply by thabaum** [reply]
@ADefWebserver I think I finally caught up to you... I wish to use this so it is a priority now for me to help best I can.  No need to reply if busy however I may have a few questions soon.

![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/13577556/32c3fa83-5ff0-4b89-8ae9-2d99ec97ea36)

**Reply by thabaum** [reply]
Not quite yet, I still have no interactivity running debug from VS.  I think this is expected potentially as the HTMLEditor.Blazor project may need to be doctored up still for .NET 8.

We can create the first issue for this going forward.

**Reply by thabaum** [reply]
interactivity working just had to set to your settings:
![image](https://github.com/ADefWebserver/ADefWebserver.Module.HtmlTextV2/assets/13577556/1b530879-bb7a-4513-bab1-79a4f2a7a8b0)
